CodeEditModules/Modules/AppPreferences/src/Model/AppPreferencesModel.swift

```diff
@@ -37,12 +37,12 @@ public final class AppPreferencesModel: ObservableObject {
     }
 
     /// Load and construct ``AppPreferences`` model from
-    /// `~/.codeedit/preferences.json`
+    /// `~/Library/Application Support/CodeEdit/preferences.json`
     private func loadPreferences() -> AppPreferences {
         if !filemanager.fileExists(atPath: preferencesURL.path) {
             let codeEditURL = filemanager
                 .homeDirectoryForCurrentUser
-                .appendingPathComponent(".codeedit", isDirectory: true)
+                .appendingPathComponent("Library/Application Support/CodeEdit", isDirectory: true)
```

Comment:
Maybe we can replace this with `baseURL`

Reply:
It looks like this is expecting a string and `baseURL` is a URL.

<img width="953" alt="image" src="https://user-images.githubusercontent.com/806104/195374742-04381794-4de0-44cb-98e4-0ff2db6f2102.png">

Reply:
`baseURL.path` would likely be the solution then. But that probably would contain the value `/User/{user}/Library/Application Support/CodeEdit`